src/Microsoft.DotNet.Darc/DarcLib/Helpers/FeedConstants.cs

```diff
@@ -11,22 +13,24 @@ public class FeedConstants
     public static readonly string MaestroManagedPublicFeedPrefix = "darc-pub";
     public static readonly string MaestroManagedInternalFeedPrefix = "darc-int";
 
-    public static readonly string MaestroManagedFeedNamePattern = @"darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)";
+    public static readonly Regex MaestroManagedFeedNamePattern = new(@"darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)", RegexOptions.Compiled);
+    public static readonly Regex MaestroManagedSymbolFeedNamePattern = new(@"darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)", RegexOptions.Compiled);
```

Comment:
```suggestion
    public static readonly Regex MaestroManagedSymbolFeedNamePattern = new(@"darc-(?<type>(int|pub))-sym-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)", RegexOptions.Compiled);
```
I think this is missing. This is what a sym feeds looks like `darc-pub-sym-dotnet-sdk-d723f1cd` vs `darc-pub-dotnet-sdk-d723f1cd`

Reply:
I reverted that change, it was a mistake. Let's just do the regexes in this PR